.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = C813
```

Comment:
C813 is stuff in _source_; so things that violate C813 would be illegal syntax in py2